### ISSUE TITLE
[Research] Enable automatic accessibility checking of content in the editor

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#3645]
### Feature Description

Many users would like to ensure that the content they publish with Pressbooks complies with WCAG accessibility standards. 

While users can use tools like Deque's [Axe Chrome browser extension](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd/related), WebAIM's [WAVE](https://wave.webaim.org/) or the University of Illinois' [Functional Accessibility Evaluator](https://fae.disability.illinois.edu/) to check the accessibility of published content, some editors and authoring platforms provide automated tools which run accessibility checks to help authors/publishers catch and fix common a11y issues _before_ publishing their content. 

In Pressbooks' context, there's a TinyMCE plugin which provides much of the functionality we'd be looking for, for example: https://www.tiny.cloud/docs/plugins/a11ychecker/

We should also investigate the use of https://github.com/campuspress/accessible-content (accessible content checker for WP editor developed by CampusPress), https://github.com/OIT-Design/ncsu-a11y-helper (AXE based WP plugin developed at NC State) and https://ryersondmp.github.io/sa11y/

### Feature Use Case

Any author could use this tool before publishing their content to identify and fix common a11y concerns. Would be of broad use/interest as an automatic guide/service to assist users in publishing more accessibly content. Many users who work with a learning management system already have some experience with tools like [Blackboard Ally](https://www.blackboard.com/accessibility/blackboard-ally.html) or Instructure's [TinyMCE-based a11y checker plugin](https://github.com/instructure/tinymce-a11y-checker) (see https://community.canvaslms.com/docs/DOC-13345-4152808104).

**Comment by MichSpieler** [comment]
An open source user [mentioned](https://github.com/pressbooks/pressbooks/issues/3325#issuecomment-1619188225) in a ticket [Editoria11y](https://wordpress.org/plugins/editoria11y-accessibility-checker/)

**Comment by SteelWagstaff** [comment]
Another user has recommended: https://accessibility-bookmarklets.org/

**Comment by SteelWagstaff** [comment]
We want to investigate the following in a development Pressbooks environment:
1. Editoria11y: https://wordpress.org/plugins/editoria11y-accessibility-checker/
2. Sa11y: https://wordpress.org/plugins/sa11y/
3. CampusPress' Accessible Content: https://github.com/campuspress/accessible-content

**Comment by SteelWagstaff** [comment]
Installed Editoria11y, Sa11y, and WP Accessibility (https://wordpress.org/plugins/wp-accessibility/) on steeldev.pressbooks.network for further testing. Added Michelle as super admin to that network as well. See https://youtu.be/9hev_6L8SPk?si=nNL97s3UrImPaokz for good introduction to why Editoria11y was created. See https://docs.google.com/document/d/1q4HUZZznjk1GHfJl1ntOc3rcsqrVZ4lQksEztgXvaaE/edit for ongoing research notes.

**Comment by SteelWagstaff** [comment]
Equalize Digital makes a premium/paid product for WordPress that might be of interest: https://equalizedigital.com/accessibility-checker/pricing/

**Comment by SteelWagstaff** [comment]
Michelle and I discussed the pros/cons of Sa11y and Editoria11y. Both seem promising and worth proceeding to tech feasibility review and beta testing with actual users. Michelle will check in with others on the customer facing team and on the community forum to identify users who have requested this functionality and might be willing to serve as initial user testers for the two tools.

**Comment by SteelWagstaff** [comment]
We gave a brief demo of Sa11y and Editora11y to the team in today's product meeting. We will be inviting network managers and authors who have expressed interest to act as beta testers over the next few weeks. Michelle will lead some user exploration & research activities, which can be captured in a separate ticket.

**Comment by MichelleWeremczuk** [comment]
Summary of accessibility checker testing with links [here](https://docs.google.com/document/d/1RbbJexVJEK-HL6zFKq2ui7L2FsJwBIGSAphd_ABx2Xw/edit?usp=sharing)
